include/oneapi/dpl/pstl/hetero/dpcpp/execution_sycl_defs.h

```diff
@@ -260,7 +274,7 @@ struct __ref_or_copy_impl<execution::device_policy<PolicyParams...>, _T>
 // Extension: hetero execution policy type trait
 template <typename _T>
 using __is_hetero_execution_policy =
-    ::std::disjunction<typename __is_device_execution_policy<_T>::type, typename __is_fpga_execution_policy<_T>::type>;
```

Comment:
As proposed @kboyarinov in offline discussion, we able rewrite this definition in more shortly way.

Reply:
Done.